swap_meet/clothing.py

```diff
@@ -1,2 +1,8 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, category="Clothing", condition=0):
+        super().__init__(category, condition)
```

Comment:
For all of the subclasses of `Item`, we could remove the `category` parameter from the child's `__init__` method since we don't want folks to change that value. We always want a `Clothing` object to have the category `"Clothing"`, so we can decide to not even let someone pass a different value.
```suggestion
    def __init__(self, condition=0):
        super().__init__("Clothing", condition)
```